diff --git a/libavutil/internal.h b/libavutil/internal.h
index 792fd29..cbcbc7b 100644
--- a/libavutil/internal.h
+++ b/libavutil/internal.h
@@ -23,6 +23,7 @@
  * common internal API header
  */
 
+#define _GNUC_SOURCE
 #ifndef AVUTIL_INTERNAL_H
 #define AVUTIL_INTERNAL_H
 
@@ -271,6 +272,13 @@ if((y)<(x)){\
     #define DECLARE_ASM_CONST(n,t,v)    static const t v
 #endif
 
+#undef HAVE_LRINT
+#undef HAVE_LLRINT
+#undef HAVE_LRINTF
+
+#define HAVE_LRINT      1
+#define HAVE_LLRINT     1
+#define HAVE_LRINTF     1
 
 #if !HAVE_LLRINT
 static av_always_inline av_const long long llrint(double x)
